lstchain/calib/camera/pixel_threshold_estimation.py

```diff
@@ -34,10 +38,10 @@ def get_threshold_from_dl1_file(dl1_path, sigma_clean):
     Function to get picture threshold from dl1 from interleaved pedestal events.
     Return modified picture threshold for tailcut cleaning method.
     Allow cleaning the most noisy pixels (for example around the star location).
-    Threshold for each pixel is define as:
+    Threshold for each pixel is defined as:
         threshold = pedestal_bias + sigma * pedestal_std.
     Recommended threshold for cleaning:
-        galactic source: picture_thresh=8, boundary_thresh=4, sigma=3
+        galactic source: picture_thresh=8, boundary_thresh=4, sigma=2.5
```

Comment:
Why did that change now?

Reply:
It was simply wrong, we use 2.5 always. Also, that should not depend on galactic or extra-gal field, since you can have a bright star in an EG field. What has to depend on the field (and this is what is intended here) is the "base" cleaning, that is what will be also applied to MC for good matching. Because in a bright field the 2.5*sigma condition would often dominate over the base cleaning we use for dark fields, hence making many "holes" in the camera.